src/components/DropZone/utils/index.ts

```diff
@@ -20,7 +20,7 @@ export function getDataTransferFiles(event: DropZoneEvent) {
       // events and uses `items` instead of `files` in this case.
       return Array.from(dt.items);
     }
-  } else if (event.target && event.target.files) {
+  } else if (isChangeEvent(event) && event.target.files) {
```

Comment:
TS was concerned that event.target may not exist because it didn't realize that at this point we must have a ChangeEvent for an HTMLInputElement - which does have a target